posixlib/src/main/scala/scala/scalanative/posix/sys/select.scala

```diff
@@ -34,15 +70,17 @@ object select {
 
 }
 
-object selectOps {
-  import select._
+object selectFdSet {
+  import select.fd_set
 
-  implicit class timevalOps(val ptr: Ptr[time.timeval]) extends AnyVal {
-    def tv_sec: time_t       = !(ptr._1)
-    def tv_usec: suseconds_t = !(ptr._2)
+  // createZeroed method _must_ be used within a Zone
 
-    def tv_sec_=(v: time_t): Unit       = !ptr._1 = v
-    def tv_usec_=(v: suseconds_t): Unit = !ptr._2 = v
-  }
+  def createZeroed()(implicit z: Zone): Ptr[fd_set] = {
+    // Be type pure and return pointer to fixed size buffer fd_set.
+    // One _could_ detect FD_SETSIZE at runtime and adapt allocated size,
+    // but the method would then be lying about the return type.
 
+    // Zone.alloc is documented as returning zeroed memory.
+    z.alloc(sizeof[fd_set]).cast[Ptr[fd_set]]
+  }
```

Comment:
What's the purpose of this method? It looks like exactly the same thing as just `alloc[fd_set]`.